R/model.R

```diff
@@ -506,6 +506,114 @@ compile_method <- function(quiet = TRUE,
 }
 CmdStanModel$set("public", name = "compile", value = compile_method)
 
+#' Check syntax of a Stan program
+#'
+#' @name model-method-syntax-check
+#' @aliases syntax_check
+#' @family CmdStanModel methods
+#'
+#' @description The `$syntax_check()` method of a [`CmdStanModel`] object
+#'   checks the Stan syntax of the model and returns `TRUE` if the compilation
+#'   succeeds. In case of a syntax error, an error is thrown.
```

Comment:
```suggestion
#' @description The `$syntax_check()` method of a [`CmdStanModel`] object
#'   checks the Stan program for syntax errors and returns `TRUE` (invisibly) if 
#'   parsing succeeds. In case of a syntax error, an error is thrown.
```